src/i18n.js

```diff
@@ -38,5 +43,15 @@ export default class I18n extends React.Component {
 I18n.propTypes = {
   locale: PropTypes.string.isRequired,
   messages: PropTypes.object.isRequired,
+  allowMissing: PropTypes.bool,
```

Comment:
Can you add `defaultProps` for the props that are not `isRequired`?

Reply:
Not sure we should set deafults for things like onMissingKey or pluralRules or interpolation. Unless we know some sane defaults to place in but I feel like node-polyglot should handle this.